packages/svelte/src/lib/createInertiaApp.ts

```diff
@@ -50,12 +52,27 @@ export default async function createInertiaApp({
   })
 
   if (isServer) {
-    const { html, head, css } = (SSR as SSRComponent).render({ id, initialPage })
+    const isSvelte5 = VERSION.startsWith('5')
+    const { html, head, css } = await (async () => {
+      if (isSvelte5) {
+        const svelteServer = await dynamicImport('svelte/server')
+        if (svelteServer && typeof svelteServer.render === 'function') {
+          return svelteServer.render(App)
+        }
+      }
 
-    return {
-      body: html,
-      head: [head, `<style data-vite-css>${css.code}</style>`],
-    }
+      return (SSR as SSRComponent).render({ id, initialPage })
+    })() as SvelteRenderResult
+
+    return css
+      ? {
+          body: html,
+          head: [head, `<style data-vite-css>${css.code}</style>`],
+        }
+      : {
+          body: `<div data-server-rendered="true" id="${id}" data-page="${escape(JSON.stringify(initialPage))}">${html}</div>`,
```

Comment:
I tried other ways to achieve this but couldn't using the `SSR` component, [this solution](https://github.com/jamesst20/inertia/blob/c8493ab9358fb520461940ef7f043987f0b39d17/packages/svelte/src/lib/createInertiaApp.ts#L52) was suggested by @jamesst20.

Reply:
Wouldn't this solution also work for Svelte 4, meaning we can get rid of SSR.svelte?

Reply:
@buhrmi That was a great idea, just tested and it totally works ✅